src/modules/data/blocks/ticket/__tests__/sagas.test.js

```diff
@@ -395,6 +407,41 @@ describe( 'Ticket Block sagas', () => {
 		} );
 	} );
 
+	describe( 'removeTicketsBlock', () => {
+		it( 'should remove tickets block', () => {
+			const gen = cloneableGenerator( sagas.removeTicketsBlock )();
+			expect( gen.next().value ).toEqual(
+				select( selectors.getSharedTicketsCount )
+			);
+
+			const clone1 = gen.clone();
+			expect( clone1.next( 2 ).done ).toEqual( true );
+
+			const clone2 = gen.clone();
```

Comment:
Why the clones? Is there an issue somewhere getting this to pass without them?

Reply:
there's an if statement at the start depending on how many shared tickets we have, so testing both cases.

Reply:
Can you make a separate `it` for that please? :)

Reply:
will do